packages/eslint-config-airbnb-base/rules/imports.js

```diff
@@ -91,7 +91,8 @@ module.exports = {
         '**/protractor.conf.js', // protractor config
         '**/protractor.conf.*.js', // protractor config
         '**/karma.conf.js', // karma config
-        '**/.eslintrc.js' // eslint config
+        '**/.eslintrc.js', // eslint config
```

Comment:
Do we want to specify that this format is called eslintrc?

```suggestion
        '**/.eslintrc.js', // eslint eslintrc config
```

Reply:
it's called "flat config", they're all eslintrc.

Reply:
2210030129